harness/determined/cli/experiment.py

```diff
@@ -17,7 +17,7 @@
 import determined.experimental
 import determined.load
 from determined import cli
-from determined.cli import checkpoint, proxy, render
+from determined.cli import _util, checkpoint, proxy, render
```

Comment:
we don't want to import directly from _util like this. we probably want to export the helper you added at the module level and import it through cli.prin...

refering to the style guide here https://github.com/rb-determined-ai/pedl/blob/9bd2168ec23fbba866194f3d7d3cfc243d267100/python_code_layout.md (@rb-determined-ai would be happy I'm finally following this 😅)

Reply:
fixed!